### ISSUE TITLE
Cumulative count/sum is confusing to work with when having more than 1 aggregate

### ISSUE THREAD

**Issue by Tony-metabase** [metabase/metabase#47531]
### Describe the bug

When you have a Cumulative count/sum having more than 1 aggregate gives the impression it defaults to a normal count when viewing as a table

### To Reproduce

1. Go to New -> Question -> Sample DB -> Products -> Cumulative Count Group by Category & Created_at Month

<img width="887" alt="image" src="https://github.com/user-attachments/assets/f7c15008-fcf2-4998-ad48-a6f1ba0751e6">

2. Visualise as a table and notice the numbers don't make sense. The Cumulative Count is simply a Count:

<img width="699" alt="image" src="https://github.com/user-attachments/assets/deb43e14-c340-4fcc-b3ac-ab83a0b6cbf2">

This is how it used to look in 49:

<img width="893" alt="image" src="https://github.com/user-attachments/assets/3faf31dd-ff03-440a-b983-468fc02905dc">

### Expected behavior

Cumulative Count needs to be more intuitive to work with

### Logs

None that are relevant

### Information about your Metabase installation

```JSON
50 and master
```

### Severity

blocking issue since questions that were working are not working as before anymore

**Comment by mngr** [comment]
Cumulative count was changed to be a window function. Now we always sort by the first breakout and partition by all the rest, so here you should just swap the breakouts
<img width="921" alt="Captura de ecrã 2024-09-02, às 19 09 58" src="https://github.com/user-attachments/assets/b8a7a236-f8c4-45ac-bce0-4f8175037d67">

**Comment by Tony-metabase** [comment]
According to this PR [#41340](https://github.com/metabase/metabase/pull/41340) this is how the cum sum/count should work:

<img width="890" alt="image" src="https://github.com/user-attachments/assets/a7921dbd-74c2-49f1-a202-073dc7e13189">

If i try to implement the same in the GUI it doesn't work 

<img width="709" alt="image" src="https://github.com/user-attachments/assets/da010d48-3b9e-446d-89c5-fecc7fd3a7f7">

and if i follow the flow you described to start with created_at and then category I end up with this:

<img width="753" alt="image" src="https://github.com/user-attachments/assets/277e2b82-94a8-4556-a939-9dc200a6aa7a">

Which is still confusing and to be able to understand what is going on I have to move the category back to the first column and order by category. Two extra steps which 

<img width="702" alt="image" src="https://github.com/user-attachments/assets/58bc6263-6f04-4fac-b277-c15516a58c95">

So in summary this PR or the spec described there is not in line to what is happening right now. It could be the pull request has an issue or the spec where changed but right now it's confusing to understand how this works. I am expecting metabase to behave like the spec

**Comment by camsaul** [comment]
This is behaving by design, the explanation in my PR was maybe a little confusing but @mngr is correct that it works as expected when you order the breakouts that way. This was the requested behavior from our customers -- see #2862

![image](https://github.com/user-attachments/assets/fbf9ab89-974f-48c0-aad0-1f20198b756f)

**Comment by mngr** [comment]
Fixed by https://github.com/metabase/metabase/pull/48028
